app/controllers/case_workers/claims_controller.rb

```diff
@@ -5,9 +5,9 @@ class CaseWorkers::ClaimsController < CaseWorkers::ApplicationController
   def index
     @claims = case tab
       when 'current'
-        current_user.claims_to_manage.submitted
+        current_user.rolable.claims.submitted
```

Comment:
can we do current_user.claims.submitted ? as both types of user have many claims?